### ISSUE TITLE
pygame.transform.skew

### ISSUE THREAD

**Issue by JiffyRob** [pygame-community/pygame-ce#2098]
**Description**
I think it would be nice if pygame offered a `pygame.transform.skew` function (or similar) that would skew an image.  eg:
![image](https://user-images.githubusercontent.com/101531330/229976278-0ba204e5-2f92-46d9-a2cd-75dc056fde85.png)
to
![luxa org-skewed-items(small)](https://user-images.githubusercontent.com/101531330/229976412-d82835ab-8e26-4bf9-a646-d07161b890b2.png)

As for arguments, probably `pygame.transform.skew(surface, axis, amount, background=None)` where `surface` is the surface to skew, `axis` is "x" or "y" for what direction to skew it in, `amount` is amount to skew it by, and `background` is the background color, defaulting to transparent.

**Comment by yunline** [comment]
You can use `_sdl2.video.Texture.draw_quad()` to get the same effect. But copying from Renderer to Surface may be time consuming.

So maybe `transform.textured_quad()` can be a better solution (?

**Comment by simplyrohan** [comment]
I agree with @yunline. Instead of just skewing, I think it would be better if you can input four points for the image to match. I think this is called a perspective warp or transformation. If you are looking for an immediate solution, look at [this](https://github.com/davidpendergast/pygame-utils/blob/main/warp.py#L51) function.

**Comment by tigercoding56** [comment]
> I agree with @yunline. Instead of just skewing, I think it would be better if you can input four points for the image to match. I think this is called a perspective warp or transformation. If you are looking for an immediate solution, look at [this](https://github.com/davidpendergast/pygame-utils/blob/main/warp.py?rgh-link-date=2023-04-05T22%3A16%3A52Z) function.

what if you need more than 4 points?
